content/Arduino Cloud/Cloud Editor/If-you-see-an-error-when-performing-an-Over-the-Air-OTA-upload.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "If you see an error when performing an Over-the-Air (OTA) upload"
+id: 15122161765916
+---
+
+Learn how to resolve errors when uploading Over-the-Air (OTA) in the Cloud Editor.
+
+Before you proceed, rule out other types of errors:
+
+* If the message on the console panel bar says **"Error Verifying \<Sketch name\>"**, it means that the sketch could not be successfully compiled. See [If your sketch doesn't compile](https://support.arduino.cc/hc/en-us/articles/4402764401554-If-your-sketch-doesn-t-compile) instead.
+* An Over-the-Air upload will only be attempted if the ![OTA symbol](img/symbol_ota.png) OTA icon is visible to the right of the Upload button. If USB upload is also available, make sure that the OTA option is selected.
+
+In this article:
+
+* [Identifying the OTA error message](#identify-error)
+* [Error reference for OTA uploads](#errors)
+
+<!--
+
+TODO: Make list of URLs with anchors
+
+#sha256unknown
+#sha256mismatch
+#otastorageinitfail
+#serverconnecterrorfail
+#otaheadercrcfail
+#otadownloadfail
+#errorwriteupdatefilefail
+#httpresponsefail
+
+-->
+
+---
+
+## Identifying the OTA error message {#identify-error}
+
+When an Over-the-Air upload fails, you can find the error in the console output.
+
+Follow these steps:
+
+1. Find the console panel. If it is hidden, show it by clicking the button in the bottom-right corner.
+2. Find the line that starts with "SHA256" or "Board error".
+3. Find the instructions for that error in the [error reference](#errors).
+
+<!-- TODO: Update image -->
+
+<img src="img/ota-error-sha.png" width=800px/>
+
+---
+
+## Error reference for OTA uploads {#errors}
+
+In this section:
+
+* [Board error - errorwriteupdatefilefail](#errorwriteupdatefilefail)
+* [Board error - httpresponsefail](#httpresponsefail)
+* [Board error - otadownloadfail](#otadownloadfail)
+* [Board error - otaheadercrcfail](#otaheadercrcfail)
+* [Board error - otastorageinitfail](#otastorageinitfail)
+* [Board error - serverconnecterrorfail](#serverconnecterrorfail)
+* [SHA256 mismatch - Unable to apply the new firmware. Previous firmware detected after restart](#sha256mismatch)
+* [SHA256 Unknown - Unknown Firmware detected after restart.](#sha256unknown)
+
+### Board error - errorwriteupdatefilefail {#errorwriteupdatefilefail}
+
+This error indicates that the device's memory partition table is improperly formatted.
+
+Follow these steps:
+
+1. Run the memory formatting sketch for your device architecture.
+1. Reinitialize the storage with this Sketch: [QSPIFormat.ino](https://github.com/arduino/ArduinoCore-mbed/blob/main/libraries/STM32H747_System/examples/QSPIFormat/QSPIFormat.ino)
```

Comment:
I believe this sketch is specific to STM32H747-based boards, i.e. GIGA/Portenta H7. Does this step only apply to these boards, or are there equivalents for other devices?